H5P.BranchedVideo/branched-video.css

```diff
@@ -14,24 +14,32 @@
   padding-left: 3%
 }
 .tapestry-play-button {
-  background: url('images/play-button.png') no-repeat scroll 0 0 transparent;
+  background: url('images/tapestry-play.png') no-repeat scroll 0 0 transparent;
```

Comment:
As per my comment in #35, please utilize image sprites for all the icons (i.e put all the icons into the same image file). See here for more details: https://www.w3schools.com/css/css_image_sprites.asp